apps/hash-graph/lib/graph/src/api/rest/entity.rs

```diff
@@ -588,6 +600,137 @@ where
     Ok(StatusCode::NO_CONTENT)
 }
 
+#[utoipa::path(
+    post,
+    path = "/entities/{entity_id}/editors/{editor}",
+    tag = "Entity",
+    params(
+        ("X-Authenticated-User-Actor-Id" = AccountId, Header, description = "The ID of the actor which is used to authorize the request"),
+        ("entity_id" = EntityId, Path, description = "The Entity to add the editor to"),
+        ("editor" = OwnedById, Path, description = "The editor to add to the entity"),
```

Comment:
Not for this PR: this is one of the places that `OwnedById` as a type is potentially confusing... dunno what to do about it though since any other type would just be an alias for this. Potentially we could rename the type to something else, and just accept that the `ownedById` field would take an `accountIdOrAccountGroupId` (or whatever)

Reply:
I agree that this is confusing. I didn't want to introduce a new type in this PR solely for this purpose but will do that in one of my follow-ups.